Generator/Sources/NeedleFramework/Generating/DependencyGraphExporter.swift

```diff
@@ -95,16 +95,23 @@ class DependencyGraphExporter {
     private func awaitSerialization(using taskHandleTuples: [(SequenceExecutionHandle<[SerializedProvider]>, String)], withTimeout timeout: TimeInterval) throws -> [SerializedProvider] {
         // Wait for all the generation to complete so we can write all the output into a single file
         var providers = [SerializedProvider]()
+        var missingDependencies = false
         for (taskHandle, componentName) in taskHandleTuples {
             do {
                 let provider = try taskHandle.await(withTimeout: timeout)
                 providers.append(contentsOf: provider)
+            } catch DependencyProviderContentError.missingDependency(let message) {
+                print(message)
```

Comment:
does calling log method also print?

Reply:
good call, I've switched this to a warning